helm/cluster/ci/ci-values.yaml

```diff
@@ -1,8 +1,202 @@
-metadata:
-  name: awesome
-  organization: giantswarm
-  description: "Awesome Giant Swarm cluster"
+global:
```

Comment:
CircleCI uses these values to render the manifests. These are also used when running `make template` locally, so you can see if rendering broke (it's like a happy path with a lot of parameters, so it will not catch all the errors).